src/stores/spaces/SpaceStore.ts

```diff
@@ -101,10 +107,18 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
     private parentMap = new EnhancedMap<string, Set<string>>();
     // Map from SpaceKey to SpaceNotificationState instance representing that space
     private notificationStateMap = new Map<SpaceKey, SpaceNotificationState>();
-    // Map from space key to Set of room IDs that should be shown as part of that space's filter
-    private spaceFilteredRooms = new Map<SpaceKey, Set<string>>(); // won't contain MetaSpace.People
-    // Map from space ID to Set of user IDs that should be shown as part of that space's filter
-    private spaceFilteredUsers = new Map<Room["roomId"], Set<string>>();
+    // Map from SpaceKey to Set of room IDs that are direct descendants of that space
+    private roomIdsBySpace: SpaceEntityMap = new Map<SpaceKey, Set<string>>(); // won't contain MetaSpace.People
+    // Map from SpaceKey to Set of space keys that are direct descendants of that space
+    private childSpacesBySpace: SpaceDescendantMap = new Map<SpaceKey, Set<SpaceKey>>();
+    // Map from SpaceKey to Set of user IDs that are direct descendants of that space
+    private userIdsBySpace: SpaceEntityMap = new Map<Room["roomId"], Set<string>>();
```

Comment:
userIds aren't tracked by-metaspace, only for real spaces, comment does not match type `Room["roomId"]`